deepchem/feat/base_classes.py

```diff
@@ -23,10 +23,14 @@ class Featurizer(object):
   new datatype.
   """
 
-  def featurize(self, datapoints: Iterable[Any],
-                log_every_n: int = 1000) -> np.ndarray:
+  def featurize(self,
```

Comment:
@walid0925 brought this up offline already so just documenting for the record. Mypy is complaining that we need to add `kwargs` on to the other subclasses so we should do that here before merging in